stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,1023 @@
+            'icon',
+            'tokenizer',
+            'dialog',
+            'bar',
+            'input',
+            'layout-grid',
+            'link'
+        ]
+    }
+};
+
+export const cozyAndCompact = () => `<div class="fd-container" style="height: 300px;">
+    <div class="fd-row">
+	<div class="fd-col fd-col--6">
+        <label class="fd-form-label">
+            Multi ComboBox Cozy Mode
+        </label>
+        <div class="fd-popover">
+            <div class="fd-popover__control" role="combobox" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
+                <div class="fd-input-group fd-input-group--control" tabindex="0">
+                    <div class="fd-tokenizer">
+                        <div class="fd-tokenizer__inner">
+                            <span class="fd-token" role="button" tabindex="0">
+                                <span class="fd-token__text">
+                                    Apple
+                                </span>
+                                <button class="fd-token__close" aria-label="unselect option: apple"></button>
+                            </span>
+                            <span class="fd-token" role="button" tabindex="0">
+                                <span class="fd-token__text">
+                                    Orange
+                                </span>
+                                <button class="fd-token__close" aria-label="close button"></button>
+                            </span>
+                            <span class="fd-token" role="button" tabindex="0">
+                                <span class="fd-token__text">
+                                    Banana
+                                </span>
+                                <button class="fd-token__close" aria-label="close button"></button>
+                            </span>
+                            <input class="fd-input fd-input-group__input fd-tokenizer__input" aria-label="multi input cozy"/>
+                        </div>
+                    </div>
+                    <span class="fd-input-group__addon fd-input-group__addon--button">
+                        <button class="fd-input-group__button fd-button fd-button--transparent" aria-label="down arrow" aria-controls="F4GcX348a" aria-expanded="false" aria-haspopup="true">
+                            <i class="sap-icon--navigation-down-arrow"></i>
+                        </button>
+                    </span>
+                </div>
+            </div>
+            <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown fd-popover__body--dropdown-fill" aria-hidden="false" id="F4GcX348a">
+            <div class="fd-popover__wrapper">
+                <ul class="fd-list fd-list--multi-input" role="listbox" aria-multiselectable="true" aria-label="selectMultipleFruitsLabel">
+                    <li class="fd-list__item is-selected" role="option" tabindex="0">
```

Comment:
See [@prsdthkr 's comment](https://github.com/SAP/fundamental-styles/pull/1802#discussion_r509599394) which still appears to be unaddressed. Will need `aria-selected=true` for all of the `is-selected` options.